build.rs

```diff
@@ -361,11 +368,20 @@ fn pregenerate_asm_main() {
         if asm_target.preassemble {
             if !std::mem::replace(&mut generated_prefix_headers, true) {
                 generate_prefix_symbols_nasm(&pregenerated, &ring_core_prefix()).unwrap();
+                // We also need to generate prefix_symbols_asm.h for Windows ARM64
+                generate_prefix_symbols_header(
+                    &pregenerated,
+                    "prefix_symbols_asm.h",
+                    '#',
+                    None,
+                    &ring_core_prefix(),
+                )
```

Comment:
If I understand correctly, there are exactly two kinds of header that suffice for all targets: One for Nasm and one for all other cases. It would be good to DRY this up so that we don't have to repeat the constants "prefix_symbols_asm.h" and `#`, and so we don't need to use a different parameter for the `prefix_condition` parameter.